Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/trace.py

```diff
@@ -181,7 +181,10 @@ def relationships(cls, commit, repository, commits_story=None):
                 if commits_story:
                     found = commits_story.by_ref.get(ref, None)
                 if not found:
-                    found = repository.find(ref)
+                    try:
+                        found = repository.find(ref)
+                    except ValueError:
+                        continue
```

Comment:
Do we want to log anything here to help in debugging later?

Reply:
I don't think so, because commits might have other valid relationships. In practice, hitting this often _does_ log something because I've seen this most often when someone does something like `Revert <rdar://1234>` which ends up triggered a log message because we fail to map `<rdar://1234>` to a commit (the line that says `...something we can't find, continuing`